docs/references/chrome-extension/create-clerk-client.mdx

```diff
@@ -3,131 +3,161 @@ title: '`createClerkClient()`'
 description: "`createClerkClient()` allows Chrome Extension developers to access a user's session information or get a token in a service worker."
 ---
 
-The Chrome Extension SDK provides a new feature for developers for interacting with a signed in user while the popup is closed. The `createClerkClient()` helper can be used in a background service worker to access a user's information or token. 
+The `createClerkClient()` helper initializes a new Clerk instance on demand and refreshes the session token if there is a valid, signed-in user. It can be used in a [background service worker](https://developer.chrome.com/docs/extensions/develop/concepts/service-workers/basics) to access a user's information or token.
 
-When the extension popup is closed, the Clerk process that normally refreshes the user's session token every 60 seconds is no longer running. The stored session will become stale, as a closed popup or a background service worker do not run in the background.
+## Why use `createClerkClient()`?
 
-The `createClerkClient()` helper wille initialize a new Clerk instance on demand, and refresh the session token is there is a valid, signed user. This is the same process as returning to a web application after the tab or browser was closed. 
+It's recommended to use `createClerkClient()` for Chrome Extension's that are configured as popups. When the popup is closed, the Clerk process that normally refreshes the user's session token every 60 seconds is no longer running, and the stored session will become stale. If a request were made 60 seconds after the popup was closed, it would fail because the session token is no longer valid. By configuring `createClerkClient()` to run in a background service worker, you can ensure that the user's session is always fresh.
```

Comment:
I would say this applies to all extensions. A user can close a side panel, or the extension may even have a side panel or popup.

Reply:
The original copy said "The Chrome Extension SDK provides a new feature for developers for interacting with a signed in user while the popup is closed." and then was written to be centered around popups.

So are you saying that this issue (Clerk not refreshing the user's session token) occurs in side panels as well?

Reply:
Yes, the original content was wrong and yes, if the side panel is closed the session is not refreshed.

Beyond that an extension doesn't have to have a popup or side panel. It can just be content scripts or service workers or content scripts+service workers or any combination of the 4 things.

I would just say that it is for any extension where the developer needs to get a token or user information in a service workers, or in a content scripts via a service worker. You can add in that when a popup or side panel are closed the session is not refreshed, so this option should be used for any action outside of a popup or side panel.